src/Castle.Windsor.Tests/Facilities/TypedFactory/TypedFactorySelectorsTestCase.cs

```diff
@@ -58,5 +58,26 @@ public void Implicitly_specified_name_falls_back_if_not_present()
 			Assert.Throws<ComponentNotFoundException>(() => factory.Create());
 
 		}
+
+		[Test, Explicit("Failing test case for issue #110")]
```

Comment:
Small nit. Can we please remove the explicit attribute? I would like these tests to fail outright in any builds.

Reply:
Sure thing, please check ed15d30.